sys/net/network_layer/ng_ndp/ng_ndp.c

```diff
@@ -681,10 +679,10 @@ static void _send_nbr_sol(kernel_pid_t iface, ng_ipv6_addr_t *tgt,
         return;
     }
 
-    LL_PREPEND(pkt, hdr);
-
     ((ng_netif_hdr_t *)hdr->data)->if_pid = iface;
 
+    LL_PREPEND(pkt, hdr);
```

Comment:
Should be none. It just looked weird to me to manipulate the headers content after it was prepended to the packet. So it's basically just a cosmetic change (as most of this PR is), ordering the LOC a little bit more thematically.